tests/Filament/TaxonomyResourceTest.php

```diff
@@ -171,6 +171,40 @@ public function test_can_delete_without_terms_in_use()
         $this->assertNull($term);
     }
 
+    public function test_can_reorder_taxonomies()
+    {
+
+        // Step 1: Create some Taxonomies
+        $taxonomy1 = Taxonomy::factory()->create();
+        $taxonomy2 = Taxonomy::factory()->create();
+        $taxonomy3 = Taxonomy::factory()->create();
+
+        // Assert initial order
+        expect($taxonomy1->fresh()->order)->toBe(1);
+        expect($taxonomy2->fresh()->order)->toBe(2);
+        expect($taxonomy3->fresh()->order)->toBe(3);
+
+        // Step 2: Reorder the terms
+        // TODO: This doesn't appear to work
```

Comment:
Remove this code and the TODOs if it doesn't work.

Reply:
This might work now that Jeremy applied a fix! The test could be updated to remove the hack (and remove the todos).